src/App.tsx

```diff
@@ -36,6 +41,26 @@ function App() {
     )
   }
 
+  const handleAddUser = (newUser: User) => {
+    const duplicateUser = users.find((user) => user.email === newUser.email)
+    if (duplicateUser) {
+      console.error('ERROR: email already in use.')
```

Comment:
Just a minor thing but how about window.alert for now? This at least will display
```suggestion
      window.alert('ERROR: email already in use.')
```